systemd/afterburn-sshkeys.target

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Synchronize afterburn-sshkeys@.service template instances
+ConditionKernelCommandLine=|ignition.platform.id=aliyun
```

Comment:
The `ConditionKernelCommandLine=...` only needs to be in the service unit rather than the target unit. 

I *think* having
```
RequiredBy=multi-user.target
```
in the target and removing it from the service is better IMO. Then:
- `multi-user.target` will start `afterburn-sshkeys.target`
- `afterburn-sshkeys.target` will start the service units
- failures on `afterburn-sshkeys@.service` will show as failed in `afterburn-sshkey.target`

Reply:
Thanks for review. That's a great idea. I will test it.